### PR TITLE
Planning: fix iterative anchoring smoother

### PR DESCRIPTION
**Pull request by JasonZhou404**

### CODE REVIEW

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc [hunk 1]
```diff
@@ -160,7 +180,7 @@ bool IterativeAnchoringSmoother::Smooth(
   const auto path_smooth_end_timestamp = std::chrono::system_clock::now();
   std::chrono::duration<double> path_smooth_diff =
       path_smooth_end_timestamp - path_smooth_start_timestamp;
-  ADEBUG << "iterative anchoring smoother totoal time: "
+  ADEBUG << "iterative anchoring path smoother totoal time: "
```
Comment (YajiaZhang) [inline modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: total

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc [hunk 2]
```diff
@@ -99,11 +99,16 @@ bool IterativeAnchoringSmoother::Smooth(
     interpolated_warm_start_path.push_back(warm_start_path.Evaluate(s));
   }
 
+  const size_t interpolated_path_size = interpolated_warm_start_path.size();
+  if (interpolated_warm_start_path < 4) {
```
Comment (xiaoxq) [inline modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: interpolated_path_size

Comment (JasonZhou404) [reply modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: fixed

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc (hunk 1 above)
Comment (JasonZhou404) [reply modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: fixed

**Pull request merged**